pallets/collection-data-feed/src/lib.rs

```diff
@@ -91,6 +94,9 @@ pub mod pallet {
 		/// The used data ID is not in the collection.
 		DataIdNotInCollection,
 
+		/// Max collection number exceeded
```

Comment:
I think this comment is copied from the other error and needs to be updated

Reply:
You're right! done